docs/privacy-policy.md

```diff
@@ -1,178 +1,157 @@
 
-When you visit our website userstyles.world (“Site”) and use our services, you trust us with your personal information. We take your privacy very seriously.
+## Introduction
 
-In this privacy notice, we describe our privacy policy. We seek to explain to you in the clearest way possible what information we store, how we use it and what rights you have in relation to it. We hope you take some time to read through it carefully, as it is important. 
+Thank you for choosing to be part of our community at UserStyles.world (“we”,
+“us”, or “our”).
 
-If there are any terms in this privacy policy that you do not agree with, please discontinue use of our site and our services.
+We are committed to protecting your personal information and your right to
+privacy. If you have any questions or concerns about our policy, or our
+practices with regards to your personal information, please contact us at
+[feedback@userstyles.world](mailto:feedback@userstyles.world).
 
-This privacy policy applies to all information strored through our Site, and/or any related services.
+When you visit our website userstyles.world (“Site”) and use our services, you
+trust us with your personal information. We take your privacy very seriously.
 
-Please read this privacy policy carefully as it will help you make informed decisions about sharing your personal information with us.
+In this privacy notice, we describe our privacy policy. We seek to explain to
+you in the clearest way possible what information we store, how we use it and
+what rights you have in relation to it. We hope you take some time to read
+through it carefully, as it is important.
 
-Table of contents:
+If there are any terms in this privacy policy that you do not agree with, please
+discontinue use of our site and our services.
 
-1. WHAT INFORMATION DO WE STORE?
-2. HOW DO WE USE YOUR INFORMATION?
-3. WILL YOUR INFORMATION BE SHARED WITH ANYONE?
-4. DO WE USE COOKIES AND OTHER TRACKING TECHNOLOGIES?
-5. HOW DO WE HANDLE YOUR SOCIAL LOGINS?
-6. IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?
-7. WHAT IS OUR STANCE ON THIRD-PARTY WEBSITES?
-8. HOW LONG DO WE KEEP YOUR INFORMATION?
-9. HOW DO WE KEEP YOUR INFORMATION SAFE?
-10. DO WE COLLECT INFORMATION FROM MINORS?
-11. WHAT ARE YOUR PRIVACY RIGHTS?
-12. DO CALIFORNIA RESIDENTS HAVE SPECIFIC PRIVACY RIGHTS?
-13. DO WE MAKE UPDATES TO THIS POLICY?
-14. HOW CAN YOU CONTACT US ABOUT THIS POLICY?
+This privacy policy applies to all information stored through our Site, and/or
+any related services. Please read it carefully as it will help you make informed
+decisions about sharing your personal information with us.
 
-## WHAT INFORMATION DO WE STORE? 
 
-### **Personal information you disclose to us**
+## What data do we collect?
 
-_In Short: We store personal information that you provide to us such as email address._
+We collect the following data:
```

Comment:
We actually have more data if you provide them to use e.g. the social media thingies.

Reply:
Maybe make it more general but still have the Username/Email Address. Account details (ex Username, email)

Reply:
Right, I'll revise it.